### DISCUSSION TITLE
Unknown number of multi-carousel

### DISCUSSION THREAD

**Discussion by colormango** [fancyapps/ui#274] [Q&A]
I need to display an unknown number of carousels on my page, how should I solve it?
const myCarousel1 = new Carousel(document.querySelector("#list-item-carousel1"), {friction: 0.8,});
const myCarousel2 = new Carousel(document.querySelector("#list-item-carousel2"), {friction: 0.8,});
...
...

**Comment by fancyapps** [comment]
Hi,

Here is one possible solution:

```js
[...document.querySelectorAll(".carousel")].forEach((el) => {
  const myCarousel = new Carousel(el, {
    // Options
  });
});
```

https://fancyapps.com/playground/2ya
